### ISSUE TITLE
Can't build or install gomplate in Go 1.13

### ISSUE THREAD

**Issue by mcandre** [hairyhenderson/gomplate#674]
Hi,

I am trying to install gomplate from source with Go 1.13. But I get an error for the YAML dependency, which appears to break syntax for go module import paths:

```console
$ go get github.com/hairyhenderson/gomplate/cmd/gomplate
go: finding github.com/hairyhenderson/gomplate v3.5.0+incompatible
go: downloading github.com/hairyhenderson/gomplate v3.5.0+incompatible
go: extracting github.com/hairyhenderson/gomplate v3.5.0+incompatible
go: downloading golang.org/x/crypto v0.0.0-20190605123033-f99c8df09eb5
go: finding github.com/spf13/cobra v0.0.5
go: downloading github.com/spf13/cobra v0.0.5
go: extracting github.com/spf13/cobra v0.0.5
go: finding github.com/spf13/afero v1.2.2
go: finding github.com/hashicorp/go-sockaddr v1.0.2
go: finding github.com/google/uuid v1.1.1
go: finding github.com/pkg/errors v0.8.1
go: finding github.com/hairyhenderson/toml v0.3.0
go: finding github.com/aws/aws-sdk-go v1.25.31
go: downloading github.com/spf13/afero v1.2.2
go: downloading github.com/hashicorp/go-sockaddr v1.0.2
go: downloading github.com/pkg/errors v0.8.1
go: downloading github.com/aws/aws-sdk-go v1.25.31
go: downloading github.com/google/uuid v1.1.1
go: extracting github.com/pkg/errors v0.8.1
go: extracting golang.org/x/crypto v0.0.0-20190605123033-f99c8df09eb5
go: downloading github.com/hairyhenderson/toml v0.3.0
go: extracting github.com/google/uuid v1.1.1
go: extracting github.com/spf13/afero v1.2.2
go: extracting github.com/hashicorp/go-sockaddr v1.0.2
go: finding github.com/docker/libkv v0.2.1
go: downloading github.com/docker/libkv v0.2.1
go: extracting github.com/docker/libkv v0.2.1
go: finding github.com/Masterminds/goutils v1.1.0
go: finding github.com/joho/godotenv v1.3.0
go: extracting github.com/hairyhenderson/toml v0.3.0
go: downloading github.com/Masterminds/goutils v1.1.0
go: downloading github.com/joho/godotenv v1.3.0
go: finding gopkg.in/hairyhenderson/yaml.v2 v2.2.2
go: extracting github.com/joho/godotenv v1.3.0
go: downloading gopkg.in/hairyhenderson/yaml.v2 v2.2.2
go: extracting github.com/Masterminds/goutils v1.1.0
go: extracting gopkg.in/hairyhenderson/yaml.v2 v2.2.2
go: finding github.com/zealic/xignore v0.3.3
go: downloading github.com/zealic/xignore v0.3.3
go: extracting github.com/zealic/xignore v0.3.3
go: finding github.com/ugorji/go/codec v1.1.7
go: finding github.com/ugorji/go v1.1.7
go: downloading github.com/ugorji/go/codec v1.1.7
go: downloading github.com/ugorji/go v1.1.7
go: extracting github.com/ugorji/go v1.1.7
go: extracting github.com/ugorji/go/codec v1.1.7
go: finding github.com/hashicorp/consul/api v1.2.0
go: finding github.com/hashicorp/consul v1.6.1
go: downloading github.com/hashicorp/consul/api v1.2.0
go: finding github.com/hashicorp/vault/api v1.0.4
go: finding github.com/hashicorp/vault v1.2.4
go: downloading github.com/hashicorp/consul v1.6.1
go: downloading github.com/hashicorp/vault v1.2.4
go: downloading github.com/hashicorp/vault/api v1.0.4
go: extracting github.com/hashicorp/consul/api v1.2.0
go: finding k8s.io/client-go v11.0.0+incompatible
go: downloading k8s.io/client-go v11.0.0+incompatible
go: extracting github.com/hashicorp/vault/api v1.0.4
go: extracting k8s.io/client-go v11.0.0+incompatible
go: finding github.com/Shopify/ejson v1.2.1
go: downloading github.com/Shopify/ejson v1.2.1
go: extracting github.com/Shopify/ejson v1.2.1
go: finding github.com/gosimple/slug v1.9.0
go: downloading github.com/gosimple/slug v1.9.0
go: extracting github.com/gosimple/slug v1.9.0
go: extracting github.com/hashicorp/vault v1.2.4
go: extracting github.com/hashicorp/consul v1.6.1
go: extracting github.com/aws/aws-sdk-go v1.25.31
go: github.com/hairyhenderson/gomplate/cmd/gomplate imports
	github.com/hairyhenderson/gomplate imports
	github.com/hairyhenderson/gomplate/data imports
	gopkg.in/hairyhenderson/yaml.v2: gopkg.in/hairyhenderson/yaml.v2@v2.2.2: parsing go.mod:
	module declares its path as: gopkg.in/yaml.v2
	        but was required as: gopkg.in/hairyhenderson/yaml.v2
```

Could the YAML dependency be fixed or replaced, so that `go get`... is able to succeed?

**Comment by hairyhenderson** [comment]
Hi @mcandre, thanks for logging this, and sorry you're seeing this error! I've fixed this particular issue in #594 and then actually migrated to go modules in #598 - though I think I'm also going to need to change the `module` name to export as `.../v3`.

I think I'm fairly close to releasing v3.6.0, so _in theory_ `go get github.com/hairyhenderson/gomplate/v3` should work after that. I'll also add a test to make sure that it works correctly in perpetuity. Stay tuned!

**Comment by mcandre** [comment]
Update:

I suspect a local proxy configuration issue may be impacting this behavior on my machine. I'll get back to you.

**Comment by hairyhenderson** [comment]
@mcandre that could be a factor, but the error you're getting is real - it was fixed in #594
